src/common/ExecutionHelper.cs

```diff
@@ -55,5 +56,26 @@ public static string PlatformSuffix
         /// a generalized reference to the platform-specific assembly.
         /// </summary>
         public static readonly string SubstitutionToken = ".{Platform}";
+
+        /// <summary>
+        /// Executes given <paramref name="action"/> with given <paramref name="culture"/>
+        /// </summary>
+        /// <param name="culture">Culture of thread to execute the <paramref name="action"/></param>
+        /// <param name="action">Action to execute with <paramref name="culture"/></param>
+        public static void ExecuteWithCulture(CultureInfo culture, Action action)
```

Comment:
Why not use [CulturedFact] instead? https://github.com/xunit/xunit/blob/master/test/test.utility/CulturedFactAttribute.cs